composer.json

```
@@ -21,10 +21,11 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
-        "symfony/framework-bundle": "~3.4|~4.0|~5.0",
+        "symfony/framework-bundle": "~3.4|~4.0|~5.0|~6.0",
         "doctrine/doctrine-bundle": "~1.0|~2.0",
         "doctrine/migrations": "^3.2"
     },
+    "minimum-stability": "dev",
```

We could try `beta` here, if we don't want a bleeding edge Psalm in our pipline. 🤔 

```suggestion
    "minimum-stability": "beta",
```

---

I believe adding `"prefer-stable": true` would help.

---

@kbond No, because that would install Symfony 5 packages on the PHP 8.0 run which is exactly not what we want.